Nodejs/Product/Nodejs/Guids.cs

```diff
@@ -53,6 +53,12 @@ internal static class Guids
         public const string OfficeToolsBootstrapperCmdSetString = "{D26C976C-8EE8-4EC4-8746-F5F7702A17C5}";
         public static readonly Guid OfficeToolsBootstrapperCmdSet = new Guid(OfficeToolsBootstrapperCmdSetString);
 
+        private const string ConnectedServicesCmdSetString = "{A114CF9C-BD45-4A48-92EF-D9BBBC0B3DF0}";
```

Comment:
The constants above seem to have the string literal inline (except the one that's public).